style/ruby/README.md

```diff
@@ -3,52 +3,75 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals). 36491dbb9
-* Avoid multiple assignments per line (`one, two = 1, 2`). #109
-* Avoid organizational comments (`# Validations`). #63
+* Avoid conditional modifiers (lines that end with conditionals). [36491dbb9]
+* Avoid multiple assignments per line (`one, two = 1, 2`). [#109]
+* Avoid organizational comments (`# Validations`). [#63]
 * Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
-  instead to emphasize code branches. 36491dbb9
-* Avoid explicit return statements. #48
-* Avoid using semicolons. #108
-* Avoid bang (!) method names. Prefer descriptive names. #122
+  instead to emphasize code branches. [36491dbb9]
+* Avoid explicit return statements. [#48]
+* Avoid using semicolons. [#108]
+* Avoid bang (!) method names. Prefer descriptive names. [#122]
 * Don't use `self` explicitly anywhere except class methods (`def self.method`)
-  and assignments (`self.attribute =`). #89
+  and assignments (`self.attribute =`). [#89]
 * Prefer nested class and module definitions over the shorthand version
-  [Example][class definition example] #332
-* Prefer `detect` over `find`. 0d819844
-* Prefer `select` over `find_all`. 0d819844
-* Prefer `map` over `collect`. 0d819844
-* Prefer `reduce` over `inject`. #237
-* Prefer double quotes for strings. #174
-* Prefer `&&` and `||` over `and` and `or`. #153
-* Prefer `!` over `not`. #195
+  [Example][class definition example] [#332]
+* Prefer `detect` over `find`. [0d819844]
+* Prefer `select` over `find_all`. [0d819844]
+* Prefer `map` over `collect`. [0d819844]
+* Prefer `reduce` over `inject`. [#237]
+* Prefer double quotes for strings. [#174]
+* Prefer `&&` and `||` over `and` and `or`. [#153]
+* Prefer `!` over `not`. [#195]
 * Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
-  calls. #183
-* Prefer `if` over `unless`. #254
-* Use `_` for unused block parameters. 0d819844
-* Prefix unused variables or parameters with underscore (`_`). #335
+  calls. [#183]
+* Prefer `if` over `unless`. [#254]
+* Use `_` for unused block parameters. [0d819844]
+* Prefix unused variables or parameters with underscore (`_`). [#335]
 * Use a leading underscore when defining instance variables for memoization.
```

Comment:
Missing square bracket around `#373`.

Reply:
I screwed this up in my conflict resolution. Thanks for finding it.